### PR TITLE
Add blob feed as a restore source

### PR DESCRIPTION
**Pull request by karajas**
@jcagme @chcosta @weshaggard

### CODE REVIEW

File: dir.props [hunk 1]
```diff
@@ -72,6 +72,7 @@
        -->
   <PropertyGroup>
     <RestoreSources Condition="'$(DotNetBuildOffline)' != 'true'">
+      https://dotnetfeed.blob.core.windows.net/dotnet-core/packages/index.json;
```
Comment (chcosta) [inline dir.props]: Is OverridePackageSource intentionally at the bottem of this list?

Comment (karajas) [reply dir.props]: I think even in the context of a full build, we would want it to be at the top. @weshaggard ?

Comment (weshaggard) [reply dir.props]: We should probably put the OverridePackageSource at the top.

**Comment by karajas** [comment]
Can I merge this in?

**Comment by chcosta** [comment]
LGTM

**Comment by karajas** [comment]
@dotnet-bot test Linux arm Release Build please

**Comment by jcagme** [comment]
LGTM

**Pull request merged**